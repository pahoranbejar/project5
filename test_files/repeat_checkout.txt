
What do you want to do?
(0) Add an item
(1) View cart
(2) Checkout
(3) Quit
Option: 0

Item search term: t

Which item do you want to add to your cart?
(0) Tomato (Roma tomato (each)) @ $0.8 (5 in stock)
(1) Salsa (16 oz mild tomato salsa) @ $2.5 (7 in stock)
(2) Tuna (8 oz can of tuna in water) @ $0.75 (11 in stock)
(3) Pepper (Orange sweet bell pepper (each)) @ $1.0 (1 in stock)
(4) Watermelon (Seedless watermelon (each)) @ $5.25 (2 in stock)
Option: 0

What do you want to do?
(0) Add an item
(1) View cart
(2) Checkout
(3) Quit
Option: 2

Tomato (Roma tomato (each)) x 1 @ $0.8 = $0.8
  Total: $0.8

Confirm purchase of $0.8? (y/n): y

What do you want to do?
(0) Add an item
(1) View cart
(2) Checkout
(3) Quit
Option: 0

Item search term: p

Which item do you want to add to your cart?
(0) Pepper (Orange sweet bell pepper (each)) @ $1.0 (1 in stock)
Option: 0

What do you want to do?
(0) Add an item
(1) View cart
(2) Checkout
(3) Quit
Option: 2

Pepper (Orange sweet bell pepper (each)) x 1 @ $1.0 = $1.0
  Total: $1.0

Confirm purchase of $1.0? (y/n): y

What do you want to do?
(0) Add an item
(1) View cart
(2) Checkout
(3) Quit
Option: 3
