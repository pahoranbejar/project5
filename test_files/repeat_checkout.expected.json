{"Beans": {"name": "Beans", "description": "1/2 lb. dry black beans", "quantity": 8, "price": 1.5}, "Rice": {"name": "Rice", "description": "1 lb. dry jasmine rice", "quantity": 10, "price": 1.0}, "Tomato": {"name": "Tomato", "description": "Roma tomato (each)", "quantity": 4, "price": 0.8}, "Avocado": {"name": "Avocado", "description": "Haas avocado (each)", "quantity": 2, "price": 1.0}, "Salsa": {"name": "Salsa", "description": "16 oz mild tomato salsa", "quantity": 7, "price": 2.5}, "Tuna": {"name": "Tuna", "description": "8 oz can of tuna in water", "quantity": 11, "price": 0.75}, "Pepper": {"name": "Pepper", "description": "Orange sweet bell pepper (each)", "quantity": 0, "price": 1.0}, "Watermelon": {"name": "Watermelon", "description": "Seedless watermelon (each)", "quantity": 2, "price": 5.25}}